docs/src/main/sphinx/release/release-379.md

```diff
@@ -0,0 +1,87 @@
+# Release 379 (27 Apr 2022)
+
+## General
+
+* Add {doc}`/connector/mariadb`. ({issue}`10046`)
+* Improve performance of queries that contain `JOIN` and `UNION` clauses. ({issue}`11935`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## Accumulo connector
+
+## Atop connector
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+## Druid connector
+
+## Elasticsearch connector
+
+## Google Sheets connector
+
+## Hive connector
+
+## Iceberg connector
+
+* Add support for {doc}`/sql/delete` with arbitrary predicates. ({issue}`11886`)
+## JMX connector
+
+## Kafka connector
+
+## Kinesis connector
+
+## Kudu connector
+
+## Local file connector
+
+## Memory connector
+
+## MongoDB connector
+
+## MySQL connector
+
+## Oracle connector
+
+## Phoenix connector
+
+## Pinot connector
+
+* Add support for Pinot 0.10. ({issue}`11475`)
```

Comment:
This removes support for 0.8.0 to 0.9.2 ... do we need to mention that?

Reply:
It still works we just dropped testing for it (because there are bugs in Pinot 0.8 which cause issues with testing). I'm fine with mentioning it but maybe we can make clear it still works but may not in future (i.e. we just don't test it).
The docs are updated though.

Reply:
I saw the doc update .. since it still work I think its fine to just ignore in the release notes then..